logic/src/commonMain/kotlin/com/wire/kalium/logic/data/asset/AssetRepository.kt

```diff
@@ -106,18 +106,19 @@ internal class AssetDataSource(
 ) : AssetRepository {
 
     override suspend fun uploadAndPersistPublicAsset(
-        mimeType: AssetType,
+        mimeType: String,
         assetDataPath: Path,
         assetDataSize: Long
     ): Either<CoreFailure, UploadedAssetId> {
         val uploadAssetData = UploadAssetData(assetDataPath, assetDataSize, mimeType, true, RetentionType.ETERNAL)
-        return uploadAndPersistAsset(uploadAssetData, assetDataPath)
+        return uploadAndPersistAsset(uploadAssetData, assetDataPath, null)
```

Comment:
Why not keeping the extension of public assets as well?

Reply:
there is no need for it since it will never be exported externally, but i will add it for unifying the behavior between private and public
 assets